changelog/unreleased/issue-2468

```diff
@@ -0,0 +1,5 @@
+Issue: Compatibility with non global azure domain
+
+Restic backups on azure only worked for storages on the global domain `core.windows.net`. This meant that backups to other domains such as Azure China (`core.chinacloudapi.cn') were not supported. Restic now allows overriding the global domain using the environment variable `AZURE_DOMAIN'.
```

Comment:
A similar description is also necessary in the Azure setup instructions in `doc/030_preparing_a_new_repo.rst`.